e2e/_suites/helm/helm_charts_test.go

```diff
@@ -122,15 +122,39 @@ func (ts *HelmChartTestSuite) aResourceWillExposePods(resourceType string) error
 		return err
 	}
 
-	endpoints := strings.SplitN(describe["Endpoints"].(string), ",", -1)
-	if len(endpoints) == 0 {
-		return fmt.Errorf("Error there are not Enpoints for the %s with the selector %s", resourceType, selector)
+	log.WithFields(log.Fields{
+		"describe":     describe,
+		"resourceType": resourceType,
+		"selector":     selector,
+	}).Debug("Describing resource")
+
+	// TO-DO: workaround meanwhile we identify why the output changed to a non-valid YAML format
```

Comment:
@kuisathaverat @jmlrt are those endpoints the same retrieved by `kubectl get endpoints -o json`? Maybe we can simplify the calculation using it instead of `kubectl describe`, which does not have a JSON/YAML output.

Reply:
@mdelapenya, sorry for the late answer. Indeed `kubectl describe endpoints <my_endpoint>` and `kubectl get endpoints <my_endpoint> -o json` provide the same details, so you can use `kubectl get` with json or yaml output.

Reply:
Thanks for you feedback @jmlrt! I addressed it using JSON format and parsing it with a dynamic JSON processor (gabs)